crypto/fipsmodule/service_indicator/service_indicator.c

```diff
@@ -92,6 +113,7 @@ uint64_t FIPS_service_indicator_after_call(void) { return 0; }
 int FIPS_service_indicator_check_approved(int before, int after) { return AWSLC_NOT_APPROVED; }
 
 void AES_verify_service_indicator(unsigned key_rounds) { }
+void AES_verify_service_indicator_with_ctx(EVP_CIPHER_CTX *ctx) { }
```

Comment:
I like this approach a lot, let the compiler optimize all these functions away when not in FIPS mode.  This should cause unused argument warnings though, I would say either suppress them (#pragma) or do a a void cast on the argument so we can build with -Werror.  I'd also add a sentence or two in code describing why these are empty functions.

Reply:
I didn't know that void casts were doable in this situation. Added :)